egs/wsj/s5/steps/nnet2/get_egs_discriminative2.sh

```diff
@@ -91,16 +86,16 @@ utils/split_data.sh $data $nj
 
 if [ $nj_ali -eq $nj ]; then
   ali_rspecifier="ark,s,cs:gunzip -c $alidir/ali.JOB.gz |"
-  all_ids=$(seq -s, $nj)
-  prior_ali_rspecifier="ark,s,cs:gunzip -c $alidir/ali.{$all_ids}.gz | copy-int-vector ark:- ark,t:- | utils/filter_scp.pl $dir/priors_uttlist | ali-to-pdf $alidir/final.mdl ark,t:- ark:- |"
+  alis=$(for all_ids in $(seq $nj); do echo $alidir/ali.$all_ids.gz; done)
```

Comment:
this variable name all_ids is not suitable now that you changed the code.  n would be better.  please make the same change elsewhere if you did the same.